content/en/blog/fsm3/index.md

```diff
@@ -0,0 +1,355 @@
+---
+date: 2024-02-06
+title: "Issuing Service"
```

Comment:
How about mentioning the bot in the title? Could it "sell" more? "Issuing Bot"/"Issuer Bot"